.github/workflows/workflow-complete.yml

```diff
@@ -0,0 +1,61 @@
+name: Workflow complete
+# This workflow is only required for externally triggered jobs that have manually
+# set the check status for a commit/PR
+
+on:
+  # Do not run untrusted code here
```

Comment:
How can we protect this script from being modified by untrusted third party? 🤔 I think these is noway?